docs/service-mesh/10-05-sidecar.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Issues with Istio sidecar injection
+type: Troubleshooting
+---
```

Comment:
TBH It's not a troubleshooting guide, if I don't have sidecar injected what should I do?

Check labels on namespace/resource level, check sidecar injector webhook components if it's running etc. Linked documents are correct, istio docs are very detailed but will be hard to find easily hints, IMO we can provide those hint here